From cd92f20428776cb12ac4867dffc4f5059ad8c76d Mon Sep 17 00:00:00 2001
From: Nicholas <Nicholas-Stone@outlook.com>
Date: Sat, 28 Apr 2018 15:14:35 +0000
Subject: [PATCH 15/16] add public/uploads to .gitignore

---
 .gitignore | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/.gitignore b/.gitignore
index 957f044..8d60278 100644
--- a/.gitignore
+++ b/.gitignore
@@ -1,6 +1,7 @@
 /node_modules
 /public/hot
 /public/storage
+/public/uploads
 /storage/*.key
 /vendor
 /.idea
@@ -15,4 +16,4 @@ yarn-error.log
 _ide_helper.php
 *.meta.php
 /config/debugbar.php
-*.txt
\ No newline at end of file
+*.txt
-- 
2.17.0

